packages/api/src/decorator/private/createDecoratorComposer.tsx

```diff
@@ -10,8 +10,8 @@ type DecoratorMiddlewareInit = typeof activityBorderDecoratorTypeName;
 
 export type DecoratorComposerComponent = (
   props: Readonly<{
-    children?: ReactNode;
-    middleware?: DecoratorMiddleware[];
+    children: ReactNode;
```

Comment:
I regretted... maybe let devs decide if they want to hide the original activity and just showing the decoration.

```suggestion
    children?: ReactNode | undefined;
```